pkgs/top-level/python-packages.nix

```diff
@@ -16363,5 +16401,28 @@ let
     };
   };
 
+  poezio = buildPythonPackage rec {
+    name = "poezio-${version}";
+    version = "0.8.1";
+
+    disabled = (!isPy3k);
+    propagatedBuildInputs = with self ; [ dnspython3 sleekxmpp ];
+
+   # The required release is not present anymore on https://github.com/fritzy/SleekXMPP/releases
+   patchPhase = ''
+     sed -i -e "s/sleekxmpp==1.2.4/sleekxmpp==1.2.5/" setup.py
+   '';
+
+    src = pkgs.fetchurl {
+      url = "http://dev.louiz.org/attachments/download/52/poezio-0.8.1.tar.xz";
+      sha256 = "0n3phh3lc82609ssfvqvd4papvhykd1sf2bm88dggh2x4mypwjff";
+    };
+
+    meta = {
+      description = "Poezio is a free console XMPP client";
```

Comment:
Do not repeat the package name in the description. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes